ccdproc/core.py

```diff
@@ -1267,8 +1298,8 @@ def cosmicray_lacosmic(ccd, sigclip=4.5, sigfrac=0.3,
         crmask, cleanarr = detect_cosmics(
                    data, inmask=None, sigclip=sigclip,
```

Comment:
was PEP8 OK with this indent? Looks double to me (and also there is a similar one, but only wit 4 spaces below). Either way make the two consistent.